### PR TITLE
Fix visualization settings for raw tables

### LINKED ISSUE #22359: Temporal breakout fields "by week" don't respect format settings in table visualization
**Describe the bug**
If you group a temporal field by week, and then view the results as a table, the temporal field values don't respect format settings. 

**To Reproduce**
1. Go to Sample Database -> Orders -> Summarize -> Created At by week
2. Go to Visualization -> Table
3. Click the gear icon next to Created At
4. Change the date style, and observe that the values in the "Created At: Week" don't change

**Expected behavior**
The values should respect the selected format settings

**Screenshots**
image

**Information about your Metabase Installation:**
Current master

**Severity**
Minor

### PR DESCRIPTION
**Pull request by ranquild**
Fixes https://github.com/metabase/metabase/issues/22359

How to test:
- Follow the steps from the issue

### CODE REVIEW

File: frontend/src/metabase/query_builder/selectors.js [hunk 1]
```diff
@@ -565,7 +565,7 @@ export const getRawSeries = createSelector(
 
     if (isShowingRawTable) {
       display = "table";
-      settings = { "table.pivot": false };
+      settings = { ...settings, "table.pivot": false };
```
Comment (ranquild) [inline frontend/src/metabase/query_builder/selectors.js:568]: What are trying to fix might be actually intended, so it seems weird to me.

**Review by kulyk — APPROVED** [review]
Looks good! I'd wait for a confirmation from @flamber, maybe we also want other viz settings to be applied in raw data too?

**Comment by ranquild** [comment]
I'll merge this because it solves the issue in way how it is originally described. I believe we should create another issue if there are different cases.

**Comment by flamber** [comment]
I was testing this, but I cannot figure out what this PR has changed.

**Comment by ranquild** [comment]
@flamber What this PR is that column settings are now used when viewing data as table. In order words, you click on `table` icon in the bottom-center of a visualization, go to Settings and change formatting settings and these would be used in the table.

**Comment by flamber** [comment]
That's not what RawData toggle is for - it should show completely without any formatting if it was up to me.

**Comment by ranquild** [comment]
Got it - I'm reverting this PR now

**Pull request merged**